diff --git a/plugin/sensible.vim b/plugin/sensible.vim
index 8563edf..6bf1df6 100644
--- a/plugin/sensible.vim
+++ b/plugin/sensible.vim
@@ -52,12 +52,7 @@ if &encoding ==# 'latin1' && has('gui_running')
   set encoding=utf-8
 endif
 
-if &listchars ==# 'eol:$'
-  set listchars=tab:>\ ,trail:-,extends:>,precedes:<,nbsp:+
-  if &termencoding ==# 'utf-8' || &encoding ==# 'utf-8'
-    let &listchars = "tab:\u21e5 ,trail:\u2423,extends:\u21c9,precedes:\u21c7,nbsp:\u00b7"
-  endif
-endif
+set listchars=tab:▸\ ,trail:-,extends:>,precedes:<,eol:¬,nbsp:.
 
 set autoread
 set fileformats+=mac
